web/faq.html

```diff
@@ -38,7 +38,7 @@
 The Mana Pool holds a <i>lot</i> of mana. Flowers like the Hydroangeas produce very small amounts of it, so you probably won't see a difference. Go get some better flowers (like Endoflames).</li><br>
 
 <li id="q10-fake"><b>I found a bug, how do I report it?</b> <a href="#q10">(permalink)</a><br>
-Report it on the <a href="https://github.com/Vazkii/Botania/issues">issue tracker</a>. Please read the <a href="https://github.com/Vazkii/Botania/blob/master/.github/CONTRIBUTING.md">contributor guidelines</a> first, though!</li><br>
+Report it on the <a href="https://github.com/Vazkii/Botania/issues">issue tracker</a>. Please read the <a href="https://github.com/VazkiiMods/Botania/blob/1.18.x/CONTRIBUTING.md">contributor guidelines</a> first, though!</li><br>
```

Comment:
VazkiiMods on the first one, and use master branch on the second one

Reply:
You missed this one

Reply:
sorry about that, fixed